src/leap/common/plugins.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# plugins.py
+# Copyright (C) 2015 LEAP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+Twisted plugins leap utilities.
+"""
+import os.path
+
+from twisted.plugin import getPlugins
+
+# A whitelist of modules from where to collect plugins dynamically.
+# For the moment restricted to leap namespace, but the idea is that we can pass
+# other "trusted" modules as options to the initialization of soledad.
+
+# TODO discover all the namespace automagically
+
+PLUGGABLE_LEAP_MODULES = ('mail', 'keymanager')
+
+rc_file = os.path.join("~", ".config", "leap", "leap.cfg")
```

Comment:
What will happen with the bundle? I guess as we don't need this file there we can just ignore it. But if one day we want to use this config from the bundle we'll need to have it in the `config` folder of the bundle.

Could you use `leap.common.config.get_path_prefix()` as we do in other parts of the code to get the config folder? This will allow us easier support for other platforms like windows, this would help us to support the bundle too but for it we'll need to know if we are `standalone` or not here.